### ISSUE TITLE
What is the meaning of samplePerSecond and time ?

### ISSUE THREAD

**Issue by Jaquier** [Microsoft/CNTK#854]
Starting minibatch loop.
 Epoch[ 1 of 25]-Minibatch[   1-  10]: MeanSqErr.MSE = 851.39990234 * 2560; time = 0.2134s; samplesPerSecond = 11994.5
 Epoch[ 1 of 25]-Minibatch[  11-  20]: MeanSqErr.MSE = 804.69541016 * 2560; time = 0.2097s; samplesPerSecond = 12206.9
 Epoch[ 1 of 25]-Minibatch[  21-  30]: MeanSqErr.MSE = 785.02363281 * 2560; time = 0.2091s; samplesPerSecond = 12243.1

**Comment by frankseide** [comment]
`time` is the time it took to compute this set of minibatches, e.g. 0.2134 seconds for minibatches 1..10; or 0.02134 s/mb. `samplesPerSecond` means how many samples were computed in a second. "Samples" are the units that minibatches consist of, think of sample vectors. If you process sequences, e.g. of words, then a sample is a single word, not an entire sequence.

For illustration, one can compute your MB size from the numbers: 11994.5 samples/sec * 0.02134 sec/mb = 255.96263 samples/mb --> minibatchSize=256.

**Comment by Jaquier** [comment]
Thank you for your quick feedback!
but I have another questions : 
I have trained Autoencoder based TIMIT sets, using HTK to get the log mel filterbank as inputs. 
All the configuration files used are downloaded from Microsoft wiki, they are stay the same, but I got the last training results based Linux is : 
**Finished Epoch[25 of 25]: [Training] MeanSqErr.MSE = 155.27413646 * 1124823;** 
totalSamplesSeen= 28120575; learningRatePerSample = 0.00039062501;
epochTime=48.3299s
CNTKCommandTrainEnd: TIMIT_TrainAutoEncoder
and the Microsoft wiki results is : 
`Finished Epoch[25 of 25]: [Training] MeanSqErr.MSE = 34.21343625 * 1124823; `
totalSamplesSeen = 28120575; learningRatePerSample = 0.00039062501; 
epochTime=11.0518s
CNTKCommandTrainEnd: TIMIT_TrainAutoEncoder

I do not know the reason , and I want to know the reason !
Hope your answers!
